### PR TITLE
creates eks

### PR DESCRIPTION
**Pull request by Mahmud-Nagasa**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.terraform
```
Comment (Mattjeppy) [inline .gitignore:1]: love that

File: S3-DynamoDB/terraform.tfstate [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "version": 4,
```
Comment (Mattjeppy) [inline S3-DynamoDB/terraform.tfstate:2]: great version choice here

File: main.tf [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+# provision ECR 
+module "ecr" {
+ source          = "./modules/ecr"
+ frontend_repository = var.frontend_repository
+ backend_repository = var.backend_repository
+}
+# Desired Size Hack
+resource "null_resource" "update_desired_size" {
```
Comment (jpsouthy93) [inline main.tf:26]: Eggcellent hack here mate

**Review by Mattjeppy — CHANGES_REQUESTED** [review]
Sort it out Mahmud, free tier only. You're going to make Eggsy84 cry.

**Pull request merged**